src/api.v2/controllers/experimentController.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/prefer-default-export */
+const config = require('../../config');
+
+const experiment = require('../model/experiment');
+const userAccess = require('../model/userAccess');
+
+const AccessRole = require('../../utils/enums/AccessRole');
+
+const { OK } = require('../../utils/responses');
+
+const createExperiment = async (req, res) => {
+  const { params: { experimentId }, user, body } = req;
+
+  const { name, description } = body;
+
+  await experiment.create({ id: experimentId, name, description });
+
+  await Promise.all([
+    userAccess.create(
+      { user_id: user.sub, experiment_id: experimentId, access_role: AccessRole.OWNER },
+    ),
+    userAccess.create(
+      { user_id: config.adminArn, experiment_id: experimentId, access_role: AccessRole.OWNER },
```

Comment:
I think `config.adminArn` is better named `config.adminSub` right?

Reply:
also I am wondering if it would make sense to store the adminArn somehwhere else (not statically in the code anyone can see). I know it shouldn't matter too much since it's not really useful on its own, but just in case.

Reply:
I agree. I was thinkign of having some kind of config server. AWS has this functionality actually: https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html